galaxy_importer/ansible_test/build_template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: {build_name}
+  labels:
+    app: automation-hub
+spec:
+  resources:
+    limits:
+      cpu: 500m
+      memory: 512Mi
+  output:
+    to:
+      kind: ImageStreamTag
+      name: {image_name}
+  source:
+    dockerfile: "FROM docker-registry.default.svc:5000/buildfactory/automation-hub-ansible-test:latest\nRUN wget '{download_url}' -O /archive/archive.tar.gz"
```

Comment:
@cutwater we have a few options on where the base image can be hosted... buildfactory project, automation-hub-jobs project, or outside OCP on quay, thoughts?

Reply:
1. For automation-hub-ansible-test I think buildfactory or quay are the best options. IIRC there were some changes that enabled quay builds for buildfactory project.

2. Can you please use multiline yaml literal to shrink line width.